seed/data/dev/01500_user_role.js

```diff
@@ -0,0 +1,22 @@
+var tableName = 'user_role'
```

Comment:
Why not filename `01400_...`?

Reply:
Must have missed that. I will change this.